modules/caddyhttp/fileserver/browsetplcontext.go

```diff
@@ -150,9 +158,14 @@ type browseTemplateContext struct {
 	// The number of files (items that aren't directories) in the listing.
 	NumFiles int `json:"num_files"`
 
-	// The total size of all files in the listing.
+	// The total size of all files in the listing. Only includes the
+	// size of the files themselves, not the size of the symlinks.
 	TotalFileSize int64 `json:"total_file_size"`
 
+	// The total size of all files in the listing, including the
+	// size of the files targeted by symlinks.
+	TotalFileSizeWithSymlinks int64 `json:"total_file_size_with_symlinks"`
```

Comment:
Without looking at the code, these comments are confusing to me. Symlinks are files, so I'm not sure which one has the size of the links themselves or which one has the size of the targets, or whether one skips them altogether.

Some docs + naming clarifications are probably in order here; maybe like `TotalFileSizeFollowSymlinks` or something.

(And what about Hard Links?)

Reply:
Since hard links use the same inode and point to the same data block, it would be correct not to count them. However, the problem is that they are difficult to identify because they are essentially just like regular files. It is unclear how to determine in which directory they should be counted and in which they should not, as they do not have additional metadata.

Reply:
Ah... I see. Ok then let's not worry about hard links.

Reply:
I'm probably out of my depth on this, feel free to make the changes you want to see 👍

Reply:
I would love to, but unfortunately, I don't write in Go 😕

Reply:
I will push some changes soon, no worries -- just renaming some things.